client/my-sites/plugins/plan-setup/index.jsx

```diff
@@ -1,81 +1,336 @@
 /**
```

Comment:
Does plugins make the most sense to describe this? `jetpack-setup`? I'm not familiar with what is being setup :)